code/modules/clothing/glasses/glasses.dm

```diff
@@ -136,6 +136,13 @@
 	item_state = "glasses"
 	vision_correction = 1 //corrects nearsightedness
 
+/obj/item/clothing/glasses/regular/jamjar
+	name = "Jamjar Glasses"
+	desc = "Also known as Virginity Protectors."
+	icon_state = "jamjar_glasses"
+	item_state = "jamjar_glasses"
+	vision_correction = 1 //corrects nearsightedness

```

Comment:
So everything downstream in a path like that has those effects? Good to know. I think that sheds some light on what's going wrong with my next experiment. Thanks.